js/modules/gomodule.go

```diff
@@ -25,10 +25,22 @@ func (gm *goModule) Instantiate(rt *sobek.Runtime) (sobek.CyclicModuleInstance,
 	if gm.exportedNames == nil {
 		named := mi.Exports().Named
 
-		gm.exportedNames = make([]string, len(named))
-		for name := range named {
-			gm.exportedNames = append(gm.exportedNames, name)
+		switch {
```

Comment:
I expect you used `switch` as there were more than one cases earlier?

Reply:
I refactored it since i think it's easier to follow when there are more than one condition int he same function, instead of a if/else/else if etc. I also believe it's more go like to work with switch instead of if/else/else if.

Reply:
I do consider it go like in the cases where:
1. it can't be rewritten as a signel if/else pair - this can
2. or when using a case can be used to check a single variable against multiple values and that is the only thing happening:

```go
s := "apple";
switch s{
case "cucumber", "pepper", "leek":
  return "vegetable";
default:
  return "not vegetable"
}
```

As in both of this case it makes it easier to read. I would argue that using a switch in other cases makes me think there is something strange going on.

I will let other people also look at this and see what they think on it - so this is not a request for a change at this point, but a nit.

Reply:
Yeah, happy to hear more feedback, before merging/reverting the switch change.

Reply:
I've reverted it back to if/else in 37a78c6cd139d3945244f8377e9278c4f33db8df